src/applications/personalization/profile360/vet360/components/AddressField/EditModal.jsx

```diff
@@ -28,17 +23,8 @@ export default class AddressEditModal extends React.Component {
     this.props.onChange(newFieldValue, field);
   }
 
-  onSubmit = () => {
-    this.props.onSubmit(expandAddress(this.props.field.value));
-  }
-
   getInitialFormValues = () => {
-    if (this.props.data) {
-      return consolidateAddress(this.props.data);
-    }
-    return {
-      countryName: 'United States',
-    };
+    return this.props.data || { countryName: 'United States' };
   }
```

Comment:
All of the deleted lines here were just leftover from the PCIU to Vet360 migration and had no effect.